vmdb/app/models/ems_amazon.rb

```diff
@@ -125,6 +125,20 @@ def vm_reboot_guest(vm, options = {})
     $log.error "MIQ(#{self.class.name}##{__method__}) vm=[#{vm.name}], error: #{err}"
   end
 
+  def stack_create(stack_name, template, options = {})
+    cloud_formation.stacks.create(stack_name, template.content, options).stack_id
+  rescue => err
+    $log.error "MIQ(#{self.class.name}##{__method__}) stack=[#{stack_name}], error: #{err}"
+    raise MiqException::MiqOrchestrationProvisionError, err.to_s
```

Comment:
I introduced manageiq specific `MiqException::MiqOrchestrationProvisionError` to generalize the error type because each provider may raise provider specific errors during stack operations.

The original error is not lost; it can be retrieved through `cause` method. I found this article http://devblog.avdi.org/2013/12/25/exception-causes-in-ruby-2-1/